send/ad_mu

```diff
@@ -2013,3 +1957,97 @@ sub haveOnlyAlumniRole {
 	my $role = shift;
 	return $role == $F_ALUMNI;
 }
+
+sub add_members_to_entry {
+
+	my $ad_entry = shift;
+	my $to_be_added = shift;
+	my $return_code = 1;
```

Comment:
This is correct, but I would prefer something like:
```
our $SUCCESS = 1;
our $FAIL = 0;

my $return_code = $SUCCESS;
```
The same in all places where you are using these return states.

Reply:
Done